reference/charm-configuration.md

```diff
@@ -0,0 +1,94 @@
+(ref-charm-configuration)=
+# Charm Configuration
+
+Since the release of Anbox Cloud 1.25.0, Anbox Cloud Charms have been updated to support the latest Ops Framework used by Juju.
```

Comment:
```suggestion
Since the release of Anbox Cloud 1.25.0, Anbox Cloud Charms have been updated to support the latest [Operator Framework](https://github.com/canonical/operator) used by Juju.
```